<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="buttonUpload">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Upload</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cardComboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string>Select card</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATtiny10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATtiny9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATtiny5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATtiny4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>32UC3A0512</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATXMEGA128A4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATXMEGA64A4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATXMEGA32A4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATXMEGA16A4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATXMEGA256A3B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATXMEGA256A3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATXMEGA192A3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATXMEGA128A3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATXMEGA64A3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATXMEGA256A1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATXMEGA192A1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATXMEGA128A1REVD</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATXMEGA128A1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATXMEGA64A1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA6450</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA3250</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA645</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA325</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATmega8U2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATmega16U2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATmega32U2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AT90USB82</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AT90USB162</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AT90USB1287</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AT90USB1286</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AT90USB647</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AT90USB646</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATmega32U4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATtiny84</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATtiny44</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATtiny24</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA128RFA1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA2561</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA2560</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA1281</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA1280</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA640</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATtiny85</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATtiny45</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATtiny25</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AT90PWM3B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AT90PWM2B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AT90PWM3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AT90PWM2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATtiny4313</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATtiny2313</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA328P</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>attiny88</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA168P</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA168</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA88P</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA88</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA48</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATTINY861</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATTINY461</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATTINY261</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATTINY26</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA8535</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA8515</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA161</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA32</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA6490</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA649</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA3290P</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA3290</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA329P</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA329</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA169</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA163</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA162</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA1284P</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA644P</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA644</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATmega324PA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA324P</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA164P</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AT90CAN32</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AT90CAN64</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AT90CAN128</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA128</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA64</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATMEGA103</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AT90S8535</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AT90S8515</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AT90S4434</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AT90S4433</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AT90S2343</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AT90S2333</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AT90S2313</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AT90S4414</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AT90S1200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATtiny15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATtiny13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATtiny12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>ATtiny11</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="buttonExit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>600</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QComboBox" name="speedComboBox">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string>Select speed</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>75</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>110</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>134</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>150</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>300</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>57600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>230400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>460800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>576000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>921600</string>
     </property>
    </item>
   </widget>
   <widget class="QTextBrowser" name="cardDialog">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>511</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>560</y>
      <width>381</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClear">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>600</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QComboBox" name="portComboBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string>Select port</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="buttonConnect">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>150</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonBuild">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>98</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Open .hex</string>
    </property>
   </widget>
   <widget class="QLabel" name="buildLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>55</y>
      <width>411</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonAbout">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>600</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>About</string>
    </property>
   </widget>
   <widget class="QLabel" name="pandaLabel">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Hi ! Welcome to PANDARDUINO, I'm Master Panda ! Nice to meet you !</string>
    </property>
   </widget>
   <widget class="QComboBox" name="textComboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <item>
     <property name="text">
      <string>LF line end</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CR line end</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>CR, LF line end</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>No line end</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hex input</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="optionsLine">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>321</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Upload
options</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lineDialog">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>430</y>
      <width>511</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>581</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>581</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <zorder>buttonUpload</zorder>
   <zorder>cardComboBox</zorder>
   <zorder>buttonExit</zorder>
   <zorder>speedComboBox</zorder>
   <zorder>cardDialog</zorder>
   <zorder>lineEdit</zorder>
   <zorder>buttonClear</zorder>
   <zorder>portComboBox</zorder>
   <zorder>buttonConnect</zorder>
   <zorder>buttonBuild</zorder>
   <zorder>buildLabel</zorder>
   <zorder>buttonAbout</zorder>
   <zorder>textComboBox</zorder>
   <zorder>optionsLine</zorder>
   <zorder>label</zorder>
   <zorder>lineDialog</zorder>
   <zorder>pandaLabel</zorder>
   <zorder>line</zorder>
   <zorder>line_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
